src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -121,11 +133,22 @@ export default class SimidMapCreative extends BaseSimidCreative {
     returnToAdButton.textContent = "Return To Ad";
     returnToAdButton.id = "returnToAd";
     returnToAdButton.onclick = () => this.playAd_(returnToAdButton); 
+    let self = this;
+    document.addEventListener('keydown', function(event) {
+      if (event.keyCode == KeyCodes.LEFT) {
```

Comment:
Same here, this doesn't seem intuitive.  As long as the buttons can be reached using tab, and you can activate them by pressing enter, you shouldn't need to add all this